package.json

```diff
@@ -131,8 +131,8 @@
     "@babel/preset-react": "^7.10.4",
     "@babel/preset-typescript": "^7.12.1",
     "@elastic/charts": "^30.2.0",
-    "@elastic/datemath": "^5.0.3",
     "@elastic/eslint-config-kibana": "^0.15.0",
+    "@opensearch/datemath": "file:./packages/opensearch-datemath",
```

Comment:
This should be named `oui-datemath`.

Reply:
@BSFishy could you check what should be the name of this package?

Reply:
Since the names we wanted are already taken, it doesn't make sense to hold this on just a simple name which is used internally only.